src/containers/User/SettingsProfile.js

```diff
@@ -103,7 +107,7 @@ export const SettingsProfile = props => {
             <Header transparent={true} navigation={props.navigation} />
             <KeyboardScrollView
               handleSubmit={handleSubmit}
-              isValid={isValid}
+              isValid={isValid && !submitPending}
```

Comment:
nit-pick: maybe we should rename the prop to something like `disabled` or `buttonEnabled`, to be more clear from outside the component?

Reply:
after trying this out, it seems that assign disabled to isValid means that the buttons are validated on page load and mean that we don't need to use any additional properties, if using disabled attribute we can use the formik .dirty property in combination with the .isValid property (depending on the form) i feel how we currently do is slightly less confusing